zkvm/src/ops.rs

```diff
@@ -298,12 +309,14 @@ impl Instruction {
 
 macro_rules! def_op {
     ($func_name:ident, $op:ident) => (
+           /// Adds a `$func_name` instruction.
            pub fn $func_name(&mut self) -> &mut Program{
              self.0.push(Instruction::$op);
              self
         }
     );
     ($func_name:ident, $op:ident, $type:ty) => (
+           /// Adds a `$func_name` instruction.
```

Comment:
nit: maybe specify "Adds a `$func_name` instruction with argument of `$type`"

Reply:
$type is too generic here, so it won't sound nice